packages/bot-web-ui/src/components/dashboard/tour-guide.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import RootStore from 'Stores/index';
+import { connect } from 'Stores/connect';
+
+type TTourGuide = {
+    label: string | boolean;
+    content: string[];
+    img?: string;
+    className: string;
+    dashboardTabIndex: number;
+    setActiveTab: (param: number) => void;
+};
+
+const TourGuide = ({ label, content, img, className, dashboardTabIndex, setActiveTab }: TTourGuide) => {
+    React.useEffect(() => {
+        setTimeout(() => {
+            setActiveTab(dashboardTabIndex);
+        }, 50);
+    }, [dashboardTabIndex]);
```

Comment:
```suggestion
    React.useEffect(() => {
        setTimeout(() => {
            setActiveTab(dashboardTabIndex);
        }, 50);
    }, [dashboardTabIndex]);
```
Just in case the `TourGuide` is suddenly re-rendered, to avoid multiple `setTimeout` running consecutively best to store a `timeout_id` and add a cleanup function to clear it. (This is unlikely though since the timeout is 50, but just to be safe 😉 )

Reply:
done ✅